packages/regl-worldview/src/commands/Triangles.js

```diff
@@ -57,12 +65,10 @@ const singleColor = (regl) =>
         return props.color;
       },
     },
-
-    // turn off depth to prevent flicker
+    // can pass in { enable: true, depth: true } to turn off depth to prevent flicker
     // because multiple items are rendered to the same z plane
-    depth: { enable: true, mask: false },
-    blend,
-
+    depth: (context, props) => props.depth || defaultDepth,
```

Comment:
the new value here stored in `commandUtils` for `defaultDepth` is actually different than the old value of `{ enable: true, mask: false }`

Reply:
As chatted offline, the default value produces visual bugs for triangles and having `mask` default to `true` fixes it.